docs/cloud-linter/getting-started.mdx

```diff
@@ -0,0 +1,32 @@
+---
+sidebar_position: 2
+sidebar_label: Getting Started
+title: Getting Started
+pagination_next: null
+description: Explore Cloud Linter and see what insights it can provide you about your cloud environment.
+hide_title: true
+---
+
+# Getting Started
+
+If you haven't already, install the momento cli. Instructions for installation and usage are available in [this repo](https://github.com/momentohq/momento-cli).
+
+Once the cli is installed, use the `cloud-linter` command to run the linter against your AWS environment. Make sure to use the AWS_PROFILE environment variable to specify the AWS account you want to use.
```

Comment:
I don't think most users are going to have profiles set up. I would change this to just say "Make sure you have AWS credentials configured" and link to their docs about the different creds options.